libs/zems/zems.zig

```diff
@@ -0,0 +1,119 @@
+pub const is_emscripten = true;
+
+const std = @import("std");
+pub const c = @cImport({
+    @cInclude("emscripten/emscripten.h");
+    @cInclude("emscripten/console.h");
+    @cInclude("emscripten/html5.h");
+    @cInclude("emscripten/emmalloc.h");
+});
+
+// Most common bindings, for others see emscripten headers included within `c`
+// https://emscripten.org/docs/api_reference/index.html
+pub const EmBool = enum(c_int) {
+    true = 1,
+    false = 0,
+};
+
+pub const AnimationFrameCallback = *const fn (time: f64, user_data: ?*anyopaque) callconv(.C) EmBool;
+
+pub inline fn requestAnimationFrame(cb: AnimationFrameCallback, user_data: ?*anyopaque) c_long {
+    const cb_ptr = @ptrCast(*const fn (time: f64, user_data: ?*anyopaque) callconv(.C) c_int, cb); // return enum doesn't pass typesafety
+    return c.emscripten_request_animation_frame(cb_ptr, user_data);
+}
+
+pub inline fn requestAnimationFrameLoop(cb: AnimationFrameCallback, user_data: ?*anyopaque) void {
+    const cb_ptr = @ptrCast(*const fn (time: f64, user_data: ?*anyopaque) callconv(.C) c_int, cb); // return enum doesn't pass typesafety
+    return c.emscripten_request_animation_frame_loop(cb_ptr, user_data);
+}
+
+pub inline fn cancelAnimationFrame(request_animation_frame_id: c_long) void {
+    c.emscripten_cancel_animation_frame(request_animation_frame_id);
+}
+
+pub inline fn sleep(ms: u32) void {
+    c.emscripten_sleep(ms);
+}
```

Comment:
I think we should just declare extern C fns instead of wrapping them. What do you think @michal-z ?